hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/HederaNativeOperations.java

```diff
@@ -199,15 +220,29 @@ default long resolveAlias(@NonNull final Bytes evmAddress) {
     ResponseCodeEnum transferWithReceiverSigCheck(
```

Comment:
Is there are a reason we left this form intact?

Reply:
This method is not actually referenced anywhere outside of tests and comments. Do you see any reason why I can't just delete it?

Reply:
I went ahead and deleted it.